### PR TITLE
feat: add complete introduction

### PR DESCRIPTION
**Pull request by Jumace**

### CODE REVIEW

File: presentations/offsite-2018/src/06--call-for-maintainers.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#Call for maintainers
+
+^I'm not sure anymore what we wanted to say here xD
```
Comment (MartinHelmut) [inline presentations/offsite-2018/src/06--call-for-maintainers.md]: Maybe something like _"you don't have to be a professional"_, _"every contribution is a god contribution"_ or _"you don't have to write code to contribute"_ ??? ^^

Comment (Jumace) [reply presentations/offsite-2018/src/06--call-for-maintainers.md]: Yeah good point, we definitely should talk about that

Comment (MartinHelmut) [reply presentations/offsite-2018/src/06--call-for-maintainers.md]: Do you want to add this (as well as the other comment) in this PR or in a separate one?

Comment (Jumace) [reply presentations/offsite-2018/src/06--call-for-maintainers.md]: I would prefer to merge it now to have a look how our slides are going together

**Pull request merged**